lib/client_success.rb

```diff
@@ -1,6 +1,5 @@
-require "active_support/core_ext/string/inflections"
-require "active_support/core_ext/hash/keys"
-require "active_support/core_ext/object/to_query"
+require "active_support"
+require "active_support/core_ext"
```

Comment:
please require only what we need from `active_support`

Reply:
Done